budgetportal/models.py

```diff
@@ -228,8 +230,6 @@ class Meta:
         ordering = ['vote_number', 'name']
 
     def save(self, *args, **kwargs):
```

Comment:
I'm sure you can totally remove this method now

Reply:
good point thanks - i've removed it